### ISSUE TITLE
PyCharm 2019.3 does not show Second Project when in full-screen.

### ISSUE THREAD

**Issue by sodre** [ChrisRM/material-theme-jetbrains#1478]
# Summary
Opening a second project in PyCharm 2019.3 does not work when Material UI is enabled *and* it is in Full-Screen.

### Expected behavior
To be able to open multiple projects opened in separate windows at the same time, without having to flip back-and-forth from full-screen.

### Actual behavior
The second window stays in an "empty canvas" mode until I take it out of MacOS full-screen mode.

### Steps to reproduce (if needed)

1. Launch PyCharm and select [Project A]
1. Put the first project in full-screen mode.
2. Open the second project by click File->Open Recent-> [Project B]
3. Select New Window
4. I get an empty canvas screen, when I expected to see the actual project.

#### Workaround:

1. Resize the second project window.
1. Set it to full-screen again.

### Environment

* OS: macOS Mojave 10.14.6
   GC: ParNew, ConcurrentMarkSweep
   Memory: 1981M
   Cores: 8
   Registry: ide.balloon.shadow.size=0
* PyCharm 2019.3 (Professional Edition)
   Build #PYA-193.5233.109, built on November 28, 2019
   Runtime version: 11.0.4+10-b520.11 x86_64
   VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
* Material Plugin version: 4.11.3 
* Non-Bundled Plugins: AWSCloudFormation, IdeaVIM, org.jetbrains.plugins.rest, aws.toolkit, com.chrisrm.idea.MaterialThemeUI, mobi.hsz.idea.gitignore, name.kropp.intellij.makefile, org.intellij.plugins.hcl, python-flake8

**Comment by mallowigi** [comment]
I've had enough of this feature. Submitting an hotfix to disable this from now on. TL;DR disable "Themed Title Bar" and restart.

**Comment by sodre** [comment]
@mallowigi, I am sorry I did not catch the report in the other posts... I was relying too much on GitHub auto-suggesting of related issues.

May I suggest shipping with it disabled (at least on the Mac) and then warning the user that this problem will happen with links to the many issues opened around that flag?

I only ask because I looked at the UI difference, and the themed title bar looks really polished!


Thanks a bunch. I know it is a ton of hard-work!
